src/app/pages/create-session/create-session.page.ts

```diff
@@ -33,8 +37,12 @@ export class CreateSessionPage implements OnInit {
       });
   }
 
-  onSubmit() {
+  async onSubmit() {
     this.form1.onSubmit();
+    let result = await this.sessionService.createSession(this.form1.myForm.value);
+    if(result){
+      this.router.navigate([`/${CommonRoutes.SESSIONS_DETAILS}`],{queryParams:{id:result._id}});
```

Comment:
here, router.back is taking user to create by me page. but in the video, user is taken to session details page. and also i have to pass id so that i can view the corresponding d=session details

Reply:
ok after creating the redirect to session details page with replace true. So that on clicking back from session details will not take back the user to create page